src/lib/FlightTasks/CMakeLists.txt

```diff
@@ -104,7 +104,10 @@ add_custom_command(
 # Create Flight Tasks Library
 ###########################################
 
-add_compile_options(-Wno-cast-align) # TODO: fix and enable
+add_compile_options(
+	-Wno-cast-align
+	-Wno-address-of-packed-member # TODO: fix in c_library_v2
```

Comment:
What's the error here? These shouldn't be touching anything in mavlink.

Reply:
You're right I got that one wrong. Removed in 992f5dcc2c.